proposals/0024-implicit-resource-binding.md

```diff
@@ -462,21 +463,9 @@ depend on the order the resources were declared, so this will help the compiler
 to preserve the order. Additionally, it will also help the compiler distinquish
 between individual resources with the same type and binding range.
 
-The constructor will call Clang builtin function
-`__builtin_hlsl_create_handle_from_implicit_binding` and will pass along all the
-arguments provided to the constructor plus the resource handle.
-
-```c++
-template <typename T> class RWBuffer<T> {
-  RWBuffer(unsigned spaceNo, int range, unsigned index, unsigned orderId, const char *name) {
-    __handle = __builtin_hlsl_resource_createhandlefromimplicitbinding(__handle, spaceNo, range, index, orderId, name);
-  }  
-}
-```
-
```

Comment:
This is described in detail in 0025-resource-initialization-and-constructors.md. There is no need to have it duplicated here.